src/test/run-promtool-unittests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+: ${CEPH_ROOT:=$SCRIPTPATH/../../}
+
+sudo docker run --rm \
+         -v "$CEPH_ROOT":/ceph \
+         --name=promtool \
+         --network=host \
+         dnanexus/promtool:2.9.2 \
+         test rules /ceph/monitoring/prometheus/alerts/test_alerts.yml
```

Comment:
i think this should not be a part of "make check" which is supposed to be composed of unit test and a very basic smoke test.

Reply:
could you put it in `qa/suites/rados/mgr` ?

Reply:
@tchaikov this is actually unit testing (it feeds the Alerting rules with fake data and test their outcomes) and does not require any Ceph code compiled and/or running. The only 'complexity' here is how to run the `promtool` binary: @aaSharma14 already tried with distro packages, but it seems that the version of `promtool` packaged in Ubuntu 16.04 doesn't support the 'test rules' option :/

The only alternative I can see here is using a Github action to directly run this container image... but I'm not too fond of adding GH actions for unit-testing...